tests/unit/backend/corpora/api_server/test_v1_collection.py

```diff
@@ -769,6 +769,8 @@ def test__get_all_collections_for_index(self):
             visibility=CollectionVisibility.PUBLIC.name,
             owner="test_user_id",
             publisher_metadata={"journal": "Nature"},
+            published_at=datetime.now(),
+            revised_at=datetime.now(),
```

Comment:
These props are no longer "required", yet the test is only _now_ asserting their value. I guess I would have expected the test change to be that setting the values to None would cause the response to exclude these props, since that's the nature of the change in the PR.

Reply:
Not really since the `null` removal isn't really something we should test, IMHO (it's good to have but it won't break the API contract). These were added now because previously the assertion basically tested two `None` values and therefore passed even without dates (which was arguably wrong). Now they would be removed from the response so the assertion would raise `KeyError` if we checked for these fields.

Reply:
Unless we're doing this on all entities consistently, I'd move it to the method impl for which it was added.